### PR TITLE
Add v1.6.0 release notes

### PR DESCRIPTION
**Pull request by alexlyp**

### CODE REVIEW

**Review by matheusd — COMMENTED** [review]
Typography for the release notes page seems pretty broken.

A quick & dirty fix to improve it a bit would be adding the following in `GetStarted.module.css`:

```css
.releaseNotesText p,
.releaseNotesText ul,
.releaseNotesText ol,
.releaseNotesText h1,
.releaseNotesText h2,
.releaseNotesText h3,
.releaseNotesText h4 {
  margin-bottom: 1em;
}
```

Also, there are some .yarncache files in the commit.

File: app/i18n/docs/en/ReleaseNotes/v1.6.0.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  * bgptr
+  * JoeGruffins
+  * fguisso
+
+## New features 
+
+  * We have added a new menu item that covers Privacy and Security tools.  
+  Users should go there to 'enable' privacy on their wallets.  This enabling
+  process is not done automatically yet, mostly due to required user
+  intervention with private passprhase entry to create needed mixed and unmixed
```
Comment (jholdstock) [inline app/i18n/docs/en/ReleaseNotes/v1.6.0.md:24]:
```suggestion
  intervention with private passphrase entry to create needed mixed and unmixed 
```

File: app/i18n/docs/en/ReleaseNotes/v1.6.0.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+  Once enabled, the privacy page will transform into an account mixer form that
+  allows users to mix funds from the unmixed account into the mixed account.  To
+  follow what dcrwallet is doing, there is a log window below.  In the future,
+  we may add better messaging that will allow updates to the mixing process
+  instead of just showing raw logs.
+
+  Once privacy is enabled we have restricted external spending to the mixed
+  account.  This is to ensure privacy is not broken by spending from any unmixed
+  account.  And we have restricted address generation to the mixed account,
```
Comment (jholdstock) [inline app/i18n/docs/en/ReleaseNotes/v1.6.0.md:36]:
```suggestion
  account.  Additionally, we have restricted address generation to the mixed account,
```

File: app/components/views/GetStartedPage/GetStarted.module.css [hunk 3]
```diff
@@ -618,6 +618,16 @@
...
+.releaseNotesText p,
+.releaseNotesText ul,
+.releaseNotesText ol,
+.releaseNotesText h1,
+.releaseNotesText h2,
+.releaseNotesText h3,
+.releaseNotesText h4 {
+  margin-bottom: 1em;
+}
+
```
Comment (jholdstock) [inline app/components/views/GetStartedPage/GetStarted.module.css:630]: To indent lists and ensure the bullets are displayed
```suggestion

.releaseNotesText ul,
.releaseNotesText ol {
  padding-left: 2em;
}
```

File: app/i18n/docs/en/ReleaseNotes/v1.6.0.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
+This release is proud to include 2 major functionality improvements for staking
+and privacy.  The past privacy improvements that were available for dcrwallet
+are now implemented in decrediton for ticket purchasing and account mixing.  
+Accountless VSP ticket purchasing has been added as well.  So no longer will
+users have to add VSPs via apikey.
```
Comment (jholdstock) [inline app/i18n/docs/en/ReleaseNotes/v1.6.0.md:5]:
```suggestion
users have to add VSPs via API key.  
```

File: app/i18n/docs/en/ReleaseNotes/v1.6.0.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+  restriction.  There is a dominant warning and users must confirm the risks
+  they are imposing by spending from unmixed accounts.
+
+  * The new accountless VSP ticket purchasing process has been added as well.
+  Now there is no need to 'link' a VSP by API key.  Users may now simply go
+  to the tickets page, select the VSP they'd like to use and the number of
+  tickets to purchase.  If privacy is enabled, the process of purchasing a 
+  ticket requires there to be a successful mix to occur. Mix sessions happen 
+  every 20 minutes and participating in a single session is usually (though not 
+  always) sufficient.  If successful the mixed split ticket funding transaction,
```
Comment (jholdstock) [inline app/i18n/docs/en/ReleaseNotes/v1.6.0.md:50]: Suggest some extra commas and a couple less words in this list:

>  If successful, the mixed split ticket funding transaction, the ticket, and the ticket's fee, should all be seen on the overview.

File: app/i18n/docs/en/ReleaseNotes/v1.6.0.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+  ticket requires there to be a successful mix to occur. Mix sessions happen 
+  every 20 minutes and participating in a single session is usually (though not 
+  always) sufficient.  If successful the mixed split ticket funding transaction, 
+  the ticket and the ticket's fee should be able to be seen on the overview.  
+  But as you may notice, the ticket's fee is not yet broadcast onto the network 
+  by the VSP until the ticket has been confirmed by 6 blocks.  If any tickets 
+  have missing or errored fees, the user will be notified if they try to close 
+  Decrediton.
+
+  * Peer count is now shown on the side bar.  This should allviate issues where
```
Comment (jholdstock) [inline app/i18n/docs/en/ReleaseNotes/v1.6.0.md:57]:
```suggestion
  * Peer count is now shown on the side bar.  This should alleviate issues where
```

**Review by jholdstock — CHANGES_REQUESTED** [review]
- Maybe unrelated to the changes in this PR - a "Newer Version" link is displayed at the top of the 1.6.0 release note, even though it is the newest.

- Some image options. I believe @behindtext and @linnutee agreed to go with the third option, and tbh I think I prefer that one too. Beware that these are ~8MB each, so probably need some compression.

1 6_mix4_30
1 6_mix555_30
1 6_mix4_distort2_30

File: app/components/views/GetStartedPage/GetStarted.module.css (hunk 3 above)
Comment (jholdstock) [reply app/components/views/GetStartedPage/GetStarted.module.css:630]: Just noticed this is likely fixed by #3033 3051

**Review by matheusd — APPROVED** [review]

File: app/components/views/GetStartedPage/GetStarted.module.css (hunk 3 above)
Comment (jholdstock) [reply app/components/views/GetStartedPage/GetStarted.module.css:633]: @ay-p I think this needs to be double-checked. It was committed into this PR, but this PR is not rebased onto #3051

The result could be a double-indentation on the lists.

**Pull request merged**